### PR TITLE
feat(localization): update translation

### PR DESCRIPTION
**Pull request by jona7150**
Updated translation using Jenkins. Please review and merge @sorenoid 

 CC: @michi6571 @masa6674

### CODE REVIEW

**Review by XuemingRocks — APPROVED** [review]
👍

File: toolkit/featureforms/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -16,18 +16,18 @@
   ~
   --><resources>
     <plurals name="enter_n_chars">
-        <item quantity="one">بيت_Enter %1$d character_____________________لاحقة</item>
-        <item quantity="other">بيت_Enter %1$d characters______________________لاحقة</item>
+        <item quantity="one">أدخل %1$d أحرف</item>
```
Comment (puneet-pdx) [inline toolkit/featureforms/src/main/res/values-ar/strings.xml:19]: 1%$d are supposed to go together for interpolating number argument. Translated into arabic should be `d$1%`

Comment (puneet-pdx) [reply toolkit/featureforms/src/main/res/values-ar/strings.xml:19]: separating them like above will not work as intended.

File: toolkit/featureforms/src/main/res/values-he/strings.xml [hunk 2]
```diff
@@ -16,18 +16,18 @@
   ~
   --><resources>
     <plurals name="enter_n_chars">
-        <item quantity="one">כן_Enter %1$d character_____________________ש</item>
-        <item quantity="other">כן_Enter %1$d characters______________________ש</item>
+        <item quantity="one">הזן תו %1$d</item>
```
Comment (puneet-pdx) [inline toolkit/featureforms/src/main/res/values-he/strings.xml:19]: `%1$d` should be together and not separated out

File: toolkit/featureforms/src/main/res/values-iw/strings.xml [hunk 3]
```diff
@@ -16,18 +16,18 @@
...
     <plurals name="enter_n_chars">
-        <item quantity="one">כן_Enter %1$d character_____________________ש</item>
-        <item quantity="other">כן_Enter %1$d characters______________________ש</item>
+        <item quantity="one">הזן תו %1$d</item>
+        <item quantity="other">הזן %1$d תווים</item>
     </plurals>
     <plurals name="value_must_be_n_characters">
-        <item quantity="one">כן_Value must be %1$d character_____________________________ש</item>
-        <item quantity="other">כן_Value must be %1$d characters______________________________ש</item>
+        <item quantity="one">הערך חייב להיות תו %1$d</item>
```
Comment (puneet-pdx) [inline toolkit/featureforms/src/main/res/values-iw/strings.xml:23]: `%1$d` should be together and not separated out

**Review by puneet-pdx — COMMENTED** [review]
There are some problems with ar, he and hr translations

File: toolkit/featureforms/src/main/res/values-ar/strings.xml (hunk 1 above)
Comment (puneet-pdx) [reply toolkit/featureforms/src/main/res/values-ar/strings.xml:19]: CC @rchintapalli @sorenoid 
 @michi6571 @masa6674 Can these issues be addressed in this PR ?

File: toolkit/featureforms/src/main/res/values-he/strings.xml (hunk 2 above)
Comment (puneet-pdx) [reply toolkit/featureforms/src/main/res/values-he/strings.xml:19]: @michi6571 @masa6674 Can these issues be addressed in this PR ?

File: toolkit/featureforms/src/main/res/values-iw/strings.xml (hunk 3 above)
Comment (puneet-pdx) [reply toolkit/featureforms/src/main/res/values-iw/strings.xml:23]: @michi6571 @masa6674 Can these issues be addressed in this PR ?

**Comment by michi6571** [comment]
> There are some problems with ar, he and iw translations

@puneet-pdx We will review. 

cc @sorenoid @MasaGithub88

**Comment by michi6571** [comment]
> > There are some problems with ar, he and iw translations
> 
> @puneet-pdx We will review.
> 
We verified that the translation looks good in Passolo LPU as screenshot below. 

image

We believe this is the known issue in the Git text viewer, as it is not RTL enabled.  If you open the xml files for RTL languages with Notepad++, you should see the correct strings as expected. 

cc @sorenoid @MasaGithub88

**Comment by puneet-pdx** [comment]
@michi6571 thanks for clarifying 👍🏽 . I tested the text in Sublime text and it looks good.

**Review by puneet-pdx — APPROVED** [review]

**Pull request merged**